lib/Doctrine/DBAL/Connection.php

```diff
@@ -1954,4 +1957,60 @@ public function ping()
             return false;
         }
     }
+
+    /**
+     * @internal
+     *
+     * @param string                                           $sql
```

Comment:
Why not use a type declaration here?

Reply:
Because of a copy/paste from some other signature.